setup.py

```diff
@@ -49,7 +49,7 @@ def run_tests(self):
     long_description_content_type='text/markdown',
     url=metadata['__url__'],
     packages=setuptools.find_packages(),
-    python_requires='>=3.6',
+    python_requires='>=3.7',
```

Comment:
Yep. We'll keep this up to date and synced with the supported versions of python. I had to drop 3.6 because the numpy dependency dropped support for python 3.6.